### PR TITLE
xcpretty: remove custom formatters and reporters.

### PR DESCRIPTION
**Pull request by barakwei**
They have been removed since they're uninteresting for now.

### CODE REVIEW

File: features/steps/formatting_steps.rb [hunk 1]
```diff
@@ -237,11 +237,6 @@
   run_xcpretty(flags)
 end
 
-When(/^I pipe to xcpretty with a custom formatter$/) do
-  formatter_path = File.expand_path('../../spec/fixtures/custom_formatter.rb', __dir__)
-  run_xcpretty("-f #{formatter_path}")
-end
-
 Then(/^I should see a custom compilation message$/) do
```
Comment (DekelAvrahami) [inline features/steps/formatting_steps.rb]: Remove.

File: features/xcpretty.feature [hunk 2]
```diff
@@ -12,3 +12,6 @@ Feature: CLI behavior
         When I run xcpretty over a big file
         Then the performance should be way faster than running cat
 
+    Scenario: Non-existing report fomar
```
Comment (DekelAvrahami) [inline features/xcpretty.feature]: `forma`->`format`

File: features/support/env.rb [hunk 3]
```diff
@@ -41,7 +41,7 @@ def run_xcpretty(flags)
   File.open(input_file.path, 'w') do |file|
     file.print run_input
   end
-  @run_output = %x(cat '#{input_file.path}' | bin/xcpretty #{flags})
+  @run_output = %x(cat '#{input_file.path}' | bin/xcpretty #{flags} 2>&1)
```
Comment (DekelAvrahami) [inline features/support/env.rb]: Why do you redirect `stderr` to `stdout`?

**Review by DekelAvrahami — CHANGES_REQUESTED** [review]
remove paragraph about custom formatters in README file. Fix and squash.

File: features/steps/formatting_steps.rb (hunk 1 above)
Comment (barakwei) [reply features/steps/formatting_steps.rb]: done

File: features/xcpretty.feature (hunk 2 above)
Comment (barakwei) [reply features/xcpretty.feature]: removed as discussed, since reports are removed anyway.

File: features/support/env.rb (hunk 3 above)
Comment (barakwei) [reply features/support/env.rb]: Not needed as the test is not removed, so this change is reverted.

File: README.md [hunk 4]
```diff
@@ -72,14 +72,6 @@ $ xcodebuild [flags] | tee xcodebuild.log | xcpretty
 
 Writing a report to a custom path can be specified using `--output PATH`.
```
Comment (DekelAvrahami) [inline README.md:74]: Remove extra space line.

File: features/support/env.rb (hunk 3 above)
Comment (DekelAvrahami) [reply features/support/env.rb]: But you didn't revert it...

**Review by DekelAvrahami — APPROVED** [review]
Fix squash and merge.

File: README.md (hunk 4 above)
Comment (barakwei) [reply README.md:74]: one

File: features/support/env.rb (hunk 3 above)
Comment (barakwei) [reply features/support/env.rb]: done

**Pull request merged**